bindings/core/src/method_handler/utils.rs

```diff
@@ -132,6 +135,24 @@ pub(crate) fn call_utils_method_internal(method: UtilsMethod) -> Result<Response
             )?;
             Response::ConflictReason(conflict)
         }
+        UtilsMethod::VerifyTransactionSyntactic {
+            transaction,
+            protocol_parameters,
+        } => {
+            TransactionPayload::try_from_dto_with_params(transaction, protocol_parameters)?;
+            Response::Ok
+        }
+        UtilsMethod::BlockBytes { block } => {
+            let block = Block::try_from_dto(block)?;
+            Response::Raw(block.pack_to_vec())
+        }
+        UtilsMethod::BlockHashWithoutNonce { block } => {
+            let block = Block::try_from_dto(block)?;
+            let block_bytes = block.pack_to_vec();
+            let hash: [u8; 32] =
+                Blake2b256::digest(&block_bytes[..block_bytes.len() - core::mem::size_of::<u64>()]).into();
+            Response::Hash(prefix_hex::encode(hash))
```

Comment:
should this be a method on `Block`? Feels a bit like too low-level logic and helpful elsewhere to have it here tbh.

Reply:
I think it's not worth it since in the pow miner we just accept bytes and not a block, so there it couldn't be used without breaking change and would probably also be less efficient since we would need to pack the bytes twice (I think)

Reply:
I agree with Alex on this one

Reply:
Changed